themes/default/content/blog/redis-cloud-provider/index.md

```diff
@@ -0,0 +1,306 @@
+
+You'll also prove the architecture works by spinning up an EC2 instance and running simple commands that target the Redis cluster.
+
+The following diagram shows the major components of the architecture:
+
+![Architecture diagram of an AWS VPC across 3 availability zones, connected to Redis Enterprise Cloud via VPC peering](architecture.png)
+
+## Project Initialization and Redis Enterprise Cloud Resources
+
+{{% notes type="info" %}}
+Before you begin this tutorial, you will need to add a payment method in the [Redis Cloud console](https://app.redislabs.com/).
+{{% /notes %}}
+
+To begin, initialize your Pulumi program in an empty directory:
+
+```bash
+pulumi new typescript
+```
+
+Then, add a reference to the Redis Cloud provider:
+
+```bash
+npm i @rediscloud/pulumi-rediscloud
+```
+
+Then, in your `index.ts` file, add use the [`getPaymentMethodOutput`](https://www.pulumi.com/registry/packages/rediscloud/api-docs/getpaymentmethod/) function to obtain your payment information, and use the resulting card ID to create a Redis Subscription:
+
+{{% notes type="info" %}}
+The Redis Cloud provider does not support Fixed subscriptions. You'll need to create a Flexible subscription in order to use the provider to manage resources with Pulumi.
+{{% /notes %}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as rediscloud from "@rediscloud/pulumi-rediscloud";
+
+const card = rediscloud.getPaymentMethodOutput({
+  cardType: "<YOUR-CARD-TYPE>" // e.g. "Visa" (capitalization matters here),
+  lastFourNumbers: "1234",
+});
+
+const subscription = new rediscloud.Subscription("redis-subscription", {
+  name: "pulumi-redis-aws-example",
+  paymentMethod: "credit-card",
+  paymentMethodId: card.id,
+  cloudProvider: {
+    regions: [
+      {
+        region: "us-east-1",
+        // This CIDR block must not conflict with the CIDR block of the AWS
+        // VPC to which you wish to connect:
+        networkingDeploymentCidr: "10.0.0.0/24",
+        // We use a single AZ here to minimize the cost incurred from running
+        // this example. For a production workload, consider multiple availability
+        // zones:
+        multipleAvailabilityZones: false,
+        preferredAvailabilityZones: ["use1-az1"],
+      },
+    ],
+  },
+
+  creationPlan: {
+    memoryLimitInGb: 10,
+    quantity: 1,
+    replication: true,
+    supportOssClusterApi: false,
+    throughputMeasurementBy: "operations-per-second",
+    throughputMeasurementValue: 20000,
+    modules: ["RedisJSON"],
+  },
+});
+```
+
+Then, create a database in your subscription:
+
+```typescript
+const database = new rediscloud.SubscriptionDatabase("redis-db", {
+  name: "my-db",
+  subscriptionId: subscription.id,
+  protocol: "redis",
+  memoryLimitInGb: 10,
+  dataPersistence: "aof-every-1-second",
+  throughputMeasurementBy: "operations-per-second",
+  throughputMeasurementValue: 20000,
+  replication: true,
+});
+```
+
```

Comment:
I know we discussed this briefly in the previous PR so I know that the example uses parameters to replace the hard-coded CC digits shown here, but I think it may be worth a quick note pointing this out.